apps/nextjs-website/src/components/Hero.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+import React from 'react';
+import { Hero as HeroEC } from '@pagopa/pagopa-editorial-components';
+import { HeroProps } from '@pagopa/pagopa-editorial-components/dist/components/Hero/index';
+import { HeroSection } from '@/lib/fetch/types/PageSection';
+
+const makeHeroProps = (props: HeroSection): HeroProps => ({
+  ...props,
+  image: props.image?.url
```

Comment:
Why do we need to add `http://localhost:1337`? I would expect Strapi to return the absolute value ready for use

Reply:
> Why do we need to add `http://localhost:1337`? I would expect Strapi to return the absolute value ready for use

This is a question for @simosalsi so I'll wait for his reply about this topic.

Reply:
If it's not possible to modify Strapi (but I suppose it can be done), then I would place it in [configuration](https://github.com/pagopa/b2b-portals/blob/main/apps/nextjs-website/src/AppEnv.ts#L6-L9) and inject it during the parsing[1^][2^].

[1^]: https://github.com/pagopa/b2b-portals/blob/main/apps/nextjs-website/src/lib/fetch/types/StrapiImage.ts
[2^]: https://github.com/pagopa/b2b-portals/blob/main/apps/nextjs-website/src/lib/fetch/navigation.ts#L43

Reply:
I opened an issue ([B2BP-297](https://pagopa.atlassian.net/browse/B2BP-297)) with the aim of solving the problem. For now, I would remove the hardcoded value and assume that Strapi returns the absolute value.